core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1093,10 +1096,15 @@ public GHResponse route( GHRequest request )
         QueryResult fromQResult = qResults.get(0);
 
         double weightLimit = request.getHints().getDouble("defaultWeightLimit", defaultWeightLimit);
+        int maxVisistedNodesForRequest = request.getHints().getInt("routing.max_visited_nodes", maxVisitedNodes);
```

Comment:
Yes we can change the name in both cases (config and url) so we do not have the `routing.` in front. I thought it would be better understandable since the number of parameters is growing.